_episodes/03-lists.md

```diff
@@ -38,20 +38,23 @@ odds are: [1, 3, 5, 7]
 ~~~
 {: .output}
 
-We select individual elements from lists by indexing them:
+We can select individual elements from lists by indexing them.
+Indexing allows us to access individual elements inside a container (such as list or string).
```

Comment:
Let's remove this line -- one introductory sentence is enough, plus let's not mention "containers"

Reply:
Done in 97af8a1